package.json

```diff
@@ -82,6 +82,7 @@
     "prettier": "^1.18.2",
     "rimraf": "^3.0.0",
     "rollup": "^1.20.3",
+    "redux-thunk": "^2.3.0",
```

Comment:
If we're going to be testing thunks, it only makes sense that this dev dependency would aid in that effort.